kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/BatchRequest.java

```diff
@@ -32,6 +32,9 @@ public class BatchRequest {
   private Map<String, String> conf;
   private List<String> args;
 
+  /** whether clean up temporary uploaded resource file in local path after execution */
+  private Boolean cleanupResourceFile = false;
```

Comment:
Seems we do not need expose this field in API, just always remove uploaded files?

Reply:
it can be an internal option for upload case.

Reply:
Agree and addressed. Extracted a dedicated method `openBatch` for the use in both `POST /bathes` and `POST /bathes/upload`  with internal control `isCleanupResourceFile` option.